client/src/main/scala/caliban/client/GraphQLResponseError.scala

```diff
@@ -17,7 +17,17 @@ case class GraphQLResponseError(
   locations: Option[List[Location]],
   path: Option[List[Either[String, Int]]],
   extensions: Option[__Value]
-)
+) {
+  def print: String =
```

Comment:
Couple of recommendations here:

1. Change name to `render`
2. Add an argument with a default value `includeExtensions = false` (And remove `fullPrint` method below)
3. Add a short scaladoc